docs/digger/installation/macos-prereqs.asciidoc

```diff
@@ -1,35 +1,30 @@
-[[macos-server]]
-=== macOS Server pre-requisites
+[[macos-prereqs]]
+==== macOS prerequisites (optional)
 
-NOTE: This section is required for building iOS applications. Skip to link:#run-digger-installer[Run Digger Installer] if you are NOT building for iOS or DO NOT wish to configure a mac machine for you digger installation.
```

Comment:
Why is this note being removed?

Reply:
i think the note (optional) contains the info in this paragraph @laurafitzgerald

Reply:
@pmacko1 the motivation of adding this is to make it clear to the user where they should go if they wish to only build Android applications. If this is being removed can we keep a link somewhere in the text to show the user. The page itself is quite big once it's built so would be difficult for the user to pick this out themselves.

Reply:
@laurafitzgerald a very good point. This is kinda overlooked when focusing on technical content. Will there be a "sections" bar in the final render? (e.g. https://docs.openshift.com/container-platform/3.7/welcome/index.html left column). I can add a TOC in the beginning of the installation doc if that will help

Reply:
@pmacko1 yes there is sections on the aerogear website when it is build. You can see this here: https://aerogear.org/docs/digger/installation/ Let's not add a TOC as it doesn't fit with the rest of the site.

Reply:
note readded and polished
resolved